### PR TITLE
download buildroot from nerves s3 bucket

### PR DESCRIPTION
**Pull request by mobileoverlord**

### CODE REVIEW

File: scripts/download-buildroot.sh [hunk 1]
```diff
@@ -35,19 +35,26 @@ rm -fr $EXTRACTED_DIRNAME $BUILDROOT_PATH
 
 if [[ ! -e "$CACHED_TARBALL_PATH" ]]; then
     # Download from Buildroot
-
+    ALTERNATIVE_URL=https://s3.amazonaws.com/nerves/build/$TARBALL_NAME
     if [[ $HASHTAG =~ 20[0-9][0-9]\.[0-1][0-9] ]]; then
         # This is an official release and is hosted on the main
         # download site.
-        DOWNLOAD_URL=https://buildroot.org/downloads/$TARBALL_NAME
+        DOWNLOAD_URL=https://foo.bar.co/$TARBALL_NAME
```
Comment (GregMefford) [inline scripts/download-buildroot.sh]: Was this line intentionally changed to try to download from https://foo.bar.co/ or was that used to test the alternate URL and accidentally committed?

Comment (mobileoverlord) [inline scripts/download-buildroot.sh]: Accidentally committed. Whoops!

**Comment by mobileoverlord** [comment]
This is ready for review. @fhunleth

**Comment by fhunleth** [comment]
Merged! Thanks!

**Pull request closed without merging**